app/models/news.rb

```diff
@@ -11,6 +11,10 @@ class News < ApplicationRecord
   validate :unable_to_be_published
   validate :news_must_be_delivered_before_now
 
+  scope :unread, ->(user) { self.count - user.news_users.count }
```

Comment:
```suggestion
  scope :unread, ->(user) { where.not(id: user.news.ids) }
```
のようにして、unreadを汎用的に使いたいです。カウントするのは使用する時につければOK。

```
def unread_news_count
      @unread_news_count = News.unread(current_user).count
end
```